src/unit_tests/wrappers/wazuh/shared/mq_op_wrappers.c

```diff
@@ -25,3 +25,12 @@ int __wrap_StartMQ(const char *path, short int type,__attribute__((unused)) shor
     check_expected(type);
     return mock();
 }
+
+/**
+ * @brief This function load the expect and will_return calls for the function StartMQ
```

Comment:
This function **loads**. Also, this doxygen comment should be in the _.h_ file instead.